terraform-sentinel/index.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "Sentinel Tutorial",
+  "description": "Writing & Testing Sentinel Policies in the Sentinel CLI and publishing to TFC",
```

Comment:
How about "Writing, Testing, and publishing Sentinel policies"? => the fact that you use the CLI and TFC seem like implementation details to me.